.github/workflows/deploy-apis-to-production.yml

```diff
@@ -1134,10 +1134,9 @@ jobs:
       - name: Update the corresponding k8s configmap(s)
         run: |
           cd src/view/
-          gcloud secrets versions access latest --secret="sta-env-view-api" > api.env
+          gcloud secrets versions access latest --secret="prod-env-view-api" > api.env
```

Comment:
[correct secret names for view and analytics](https://github.com/airqo-platform/AirQo-api/pull/1062/commits/e1fa6cb672baebaf9a00c6aa2ffe5f6e721ba57c)